### PR TITLE
Handle .env as special data

### PR DESCRIPTION
**Pull request by imranolas**
In order to template env files in the same manner as other special dot files

### CODE REVIEW

File: lib/init.js [hunk 1]
```diff
@@ -79,6 +79,6 @@ module.exports = function (opts, callback) {
       );
     }
 
-    callback(err);
+    callback(err, results);
```
Comment (ryan-roemer) [inline lib/init.js:82]: Where is this used?

**Comment by ryan-roemer** [comment]
Please update readme and add at least an assert for the new template variable in the tests. Thanks!

**Comment by ryan-roemer** [comment]
What happens if you have a real ".env" file in the templates base directory? (We're looking for "something in the templates module breaks" type things...)

File: lib/init.js (hunk 1 above)
Comment (imranolas) [reply lib/init.js:82]: It's not used yet but the function signature in the doc string does declare it. 
https://github.com/formidablelabs/denim/blob/master/lib/init.js#L21

Also, if you're building tooling on top of denim (as we are with the dashboard) it's useful to expose the results for any additional processing.

Comment (ryan-roemer) [reply lib/init.js:82]: Let's revert for now and update the doc string. It's internal API that is unused.

**Comment by ryan-roemer** [comment]
Let's close this in favor of https://github.com/FormidableLabs/denim/pull/5 which can generically handle gitignored-but-I-still-want-it files.

**Pull request closed without merging**